http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/MonitorAtBoot.scala

```diff
@@ -36,8 +37,24 @@ class MonitorAtBoot[F[_]](implicit F: Async[F],
                     statusOpt <- clusterQuery.getClusterStatus(c.id).transaction
                     tid <- traceId.ask
                     s <- F.fromEither(
-                      statusOpt.toRight(new Exception(s"${c.id} not found after transition. This is very weird!"))
+                      statusOpt
+                        .toRight(new Exception(s"${tid} | ${c.id} not found after transition. This is very weird!"))
                     )
+                    now <- nowInstant[F]
+                    _ <- if (s != RuntimeStatus.Running) {
+                      // There's slight chance where pubsub message is never published during a redeploy.
```

Comment:
In this case, wouldn't there be a row in the DB with the patch details? As we were talking about yesterday, maybe this can be part of a series of DB checks after redeploy (until we have no-downtime deployments)

Reply:
I didn't think we persist target machine type...just checked database, seems there is `masterMachineType`, is that the target machineType? and we only have patchInProgress when we need to change machine type?

Reply:
Yeah, that's the target machine type. If `patchInProgress` is true for a row in the DB but there's no `UpdateRuntimeMessage` in the queue after redeploy, then we should be able to make a new message for the queue using the `masterMachineType` in the `PatchTable`

Reply:
Ah but this might only be for updates that need a stop-start.. @jdcanas do you know?